iotilecore/version.py

```diff
@@ -1 +1 @@
-version = "5.0.0"
+version = "5.0.1"
```

Comment:
Marking a comment in the review to make sure we answer the question justifying a patch release now, so that we can have practice with the process.

Reply:
Thanks for flagging the process here Matt.  In this case I believe immediate release is justified because the bug is hit by an external customer and it's impact is particularly bad.  It results in all data uploaded to the cloud being wrong...

Reply:
Sounds good. Definitely a fair justification and now we have it recorded here for posterity :)